README.md

```diff
@@ -1 +1,14 @@
-# utils
\ No newline at end of file
+# utils
+***
+## validateCredential
+This method can be used to validate api_key. To use this, get utils module in project and import validateCredential and use it as a middleware in api.
+
+Example:
+```
+const validateCredential = require('./utils/validateCredential');
```

Comment:
Users should be accessing using first doing `npm install exponential-utils` or `npm install @exponential/utils` and then here, using `require('exponential-utils')`. We should be publishing the utils module using npm publish publicly. Let me know if you need help in doing that.